### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by HStakhiv**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ):
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list):
```
Comment (FranchukS) [inline app/main.py:21]: Add annotation for return value in all methods:
def serve_cars(self, cars: list) -> 'value':

**Review by FranchukS — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (HStakhiv) [reply app/main.py]: Done!

**Comment by HStakhiv** [comment]
Is everything good now?

**Review by FranchukS — APPROVED** [review]
As for me, looks good)

**Review by Y-Havryliv — APPROVED** [review]
GJ!

**Pull request closed without merging**